File: app/src/main/java/com/asmat/rolando/popularmovies/database/FavoriteMovie.java
package com.asmat.rolando.popularmovies.database;

import android.arch.persistence.room.*;

@Entity(tableName = "favorite_movies",
        foreignKeys = @ForeignKey(
                entity = Movie.class,
                parentColumns = "id",
                childColumns = "id"))

public class FavoriteMovie {
    @PrimaryKey
    public int id;

}

File: app/src/main/java/com/asmat/rolando/popularmovies/database/entities/FavoriteMovie.kt
package com.asmat.rolando.popularmovies.database.entities

import android.arch.persistence.room.*

@Entity(tableName = "favorite_movies")
data class FavoriteMovie(@ColumnInfo(name = "poster_path")
                         val posterPath: String?,
                         val overview: String,
                         @ColumnInfo(name = "release_date")
                         val releaseDate: String,
                         @PrimaryKey val id: Int,
                         val title: String,
                         @ColumnInfo(name = "backdrop_path")
                         val backdropPath: String?,
                         @ColumnInfo(name = "vote_average")
                         val voteAverage: Double)